main.go

```diff
@@ -4,25 +4,38 @@ import (
 	"context"
 	"embed"
 
+	"github.com/knadh/koanf/v2"
 	"github.com/wailsapp/wails/v2"
 	"github.com/wailsapp/wails/v2/pkg/options"
 	"github.com/wailsapp/wails/v2/pkg/options/assetserver"
 
+	"github.com/kmesiab/orion/config"
 	gh "github.com/kmesiab/orion/github"
 )
 
 //go:embed all:frontend/dist
 var assets embed.FS
 
+// k is the global config instance
+// This is used to hold the configuration
+var k = koanf.New(".")
+
 func main() {
+	// Load the configuration file
+	config, err := config.LoadConfig(k, "env.json")
+	if err != nil {
+		println("Error loading config, set to defaults", err.Error())
+	}
```

Comment:
Good idea on configurable file path through command-line arguments! We want to avoid env variables. 
Also as to your second point, I am currently not exiting the application as if LoadConfig fails to load a file path it falls back to the default values, therefore do not want to exit the application in this case. However we can consider both options